src/main/java/com/mraof/minestuck/item/weapon/BaguetteWeaponItem.java

```diff
@@ -0,0 +1,47 @@
+package com.mraof.minestuck.item.weapon;
+
+import com.mraof.minestuck.item.MSItems;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.item.ItemEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.IItemTier;
+import net.minecraft.item.ItemStack;
+import net.minecraft.potion.EffectInstance;
+import net.minecraft.potion.Effects;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraft.util.text.TranslationTextComponent;
+import net.minecraft.world.World;
+
+import java.util.Random;
+
+public class BaguetteWeaponItem extends WeaponItem
+{
+    public BaguetteWeaponItem(IItemTier tier, int attackDamageIn, float attackSpeedIn, float efficiency, MSToolType toolType, Properties builder)
+    {
+        super(tier, attackDamageIn, attackSpeedIn, efficiency, toolType, builder);
+    }
+    
+    @Override
+    public boolean hitEntity(ItemStack itemStack, LivingEntity target, LivingEntity attacker)
+    {
+        ItemStack crumbs = new ItemStack(MSItems.BREADCRUMBS, 1);
```

Comment:
Two minor suggestions:
- Create the item stack in the if-statement since that's where it's used
- Would it make sense to have a chance less than 100% of dropping breadcrumbs on a hit?